CHANGELOG.md

```diff
@@ -76,6 +76,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 * Changed `compas_rhino.objects.get_polyline_coordinates` to deprecated (removed in v2.3).
 * Changed `compas_rhino.objects.get_polygon_coordinates` to deprecated (removed in v2.3).
 * Fixed a bug in `worldtransformation` of `compas.scene.SceneObject` to include the object's own frame.
+* `compas_rhino.install` with no version specified installs to all available versions of Rhino.
+* `compas_rhino.install` installs to IronPython, IronPython (Old) and CPython environments.
```

Comment:
could you make an overview of what is done by the install command for different versions and scenarios?

* the phrasing seems to suggest that all these installations are attempted simultaneously.
* i have not had much success with symlinking and the CPython environment (i think we should promote using `pip` in that case)

Reply:
* having packages installed both in the scripts folder and in the site packages folder breaks the new code editor
* in the new script editor, users should use only one technique consistently (`# r: compas, ...` v. `[rhinopython] -m pip install compas ...`) because the installed packages end up in different locations and will again clash

Reply:
No version specified

```
(compas-dev) C:\Users\ckasirer\repos\compas>python -m compas_rhino.install 
Install folder for Rhino version 5.0 not found: The scripts folder does not exist in this location: C:\Users\ckasirer\AppData\Roaming\McNeel\Rhinoceros\5.0\scripts
Install folder for Rhino version 6.0 not found: The scripts folder does not exist in this location: C:\Users\ckasirer\AppData\Roaming\McNeel\Rhinoceros\6.0\scripts

Installing COMPAS packages to Rhino 7.0 scripts folder:
C:\Users\ckasirer\AppData\Roaming\McNeel\Rhinoceros\7.0\scripts

   compas               OK
   compas_ghpython      OK
   compas_model         OK
   compas_rhino         OK
   compas_timber        OK
   compas_bootstrapper  OK

Running post-installation steps...

   compas_ghpython      OK: Installed 5 GH User Objects
   compas_timber        OK: Installed 31 GH User Objects

Install completed.

Installing COMPAS packages to Rhino 8.0 scripts folder:
C:\Users\ckasirer/.rhinocode/py27-rh8/Lib/site-packages

   compas               OK
   compas_ghpython      OK
   compas_model         OK
   compas_rhino         OK
   compas_timber        OK
   compas_bootstrapper  OK

Running post-installation steps...

   compas_ghpython      OK: Installed 5 GH User Objects
   compas_timber        OK: Installed 31 GH User Objects

Install completed.

Installing COMPAS packages to Rhino 8.0 scripts folder:
C:\Users\ckasirer/AppData/Roaming/McNeel/Rhinoceros/8.0/scripts

   compas               OK
   compas_ghpython      OK
   compas_model         OK
   compas_rhino         OK
   compas_timber        OK
   compas_bootstrapper  OK

Running post-installation steps...

   compas_ghpython      OK: Installed 5 GH User Objects
   compas_timber        OK: Installed 31 GH User Objects

Install completed.
```

Only Rhino 7

```
(compas-dev) C:\Users\ckasirer\repos\compas>python -m compas_rhino.install -v7.0

Installing COMPAS packages to Rhino 7.0 scripts folder:
C:\Users\ckasirer\AppData\Roaming\McNeel\Rhinoceros\7.0\scripts

   compas               OK
   compas_ghpython      OK
   compas_model         OK
   compas_rhino         OK
   compas_timber        OK
   compas_bootstrapper  OK

Running post-installation steps...

   compas_ghpython      OK: Installed 5 GH User Objects
   compas_timber        OK: Installed 31 GH User Objects

Install completed.
```

Only Rhino8

```
(compas-dev) C:\Users\ckasirer\repos\compas>python -m compas_rhino.install -v8.0 

Installing COMPAS packages to Rhino 8.0 scripts folder:
C:\Users\ckasirer/.rhinocode/py27-rh8/Lib/site-packages

   compas               OK
   compas_ghpython      OK
   compas_model         OK
   compas_rhino         OK
   compas_timber        OK
   compas_bootstrapper  OK

Running post-installation steps...

   compas_ghpython      OK: Installed 5 GH User Objects
   compas_timber        OK: Installed 31 GH User Objects

Install completed.

Installing COMPAS packages to Rhino 8.0 scripts folder:
C:\Users\ckasirer/AppData/Roaming/McNeel/Rhinoceros/8.0/scripts

   compas               OK
   compas_ghpython      OK
   compas_model         OK
   compas_rhino         OK
   compas_timber        OK
   compas_bootstrapper  OK

Running post-installation steps...

   compas_ghpython      OK: Installed 5 GH User Objects
   compas_timber        OK: Installed 31 GH User Objects

Install completed.
```